doc/source/getting_started/quick_start/index.rst

```diff
@@ -68,6 +68,8 @@ To quickly install Covalent and run a short demo, follow the four steps below.
 
       # Dispatch the workflow
       dispatch_id = ct.dispatch(workflow)(1, 2)
+      result = ct.get_result(dispatch_id)
```

Comment:
`result = ct.get_result(dispatch_id,wait=True)`, since get_result is an asynchronous call, it will immediately return the partial result. `wait=True` forces the get result to wait until the job is completed.

Reply:
Good point, I forgot about that. It was set correctly in a different example.